src/js/components/RangeInput/index.d.ts

```diff
@@ -16,11 +16,15 @@ export interface RangeInputProps {
   step?: number;
   color?: ColorType | ColorInterface[];
   value?: number | string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
 }
 
 export interface RangeInputExtendedProps
   extends RangeInputProps,
-    Omit<JSX.IntrinsicElements['input'], 'color' | 'step' | 'value'> {}
+    Omit<
+      JSX.IntrinsicElements['input'],
+      'color' | 'step' | 'value' | 'onChange'
```

Comment:
What is the difference between the default `onChange` that is passed though the intrinsic element and the `onChange` type defined on line 19?